src/Esfa.Vacancy.UnitTests/CreateApprenticeship/Application/GivenACreateApprenticeshipCommandHandler/WhenHandlingACommand.cs

```diff
@@ -77,23 +88,49 @@ public void AndCommandNotValid_ThenThrowsValidationException()
         [Test]
         public void ThenValidatesRequest()
         {
-            _mockValidator.Verify(validator => 
+            _mockValidator.Verify(validator =>
                 validator.ValidateAsync(_validRequest, It.IsAny<CancellationToken>()),
                 Times.Once);
         }
 
+        [Test]
+        public void ThenFetchVacancyOwnerLink()
+        {
+            _mockVacancyOwnerService.Verify(svc => svc.GetEmployersInformationAsync(
+                It.IsAny<int>(), It.IsAny<int>(), It.IsAny<int>()));
+        }
+
+        [Test]
+        public void AndIfLinkIsNotRetrieved_ThenThrowUnauthorisedException()
+        {
+            var validRequest = _fixture.Create<CreateApprenticeshipRequest>();
```

Comment:
you should be able to get away with using the existing _validRequest and then not need to re-setup the validator.